src/components/organisms/Navbar/Navbar/Navbar.tsx

```diff
@@ -1,84 +1,199 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
+import Headroom from 'react-headroom';
+import { faBars } from '@fortawesome/free-solid-svg-icons';
 
 import { colors } from '../../../../constants/colors';
-import { navbarHeight } from '../../../../constants/components';
-import useScrollThreshold from '../useScrollThreshold/useScrollThreshold';
-import FlexContainer from '../../../layout/FlexContainer/FlexContainer';
+import { breakpoints } from '../../../../constants/breakpoints';
+import { spacing } from '../../../../constants/spacing';
+import { navbarHeight, mobileNavbarHeight } from '../../../../constants/components';
+import { useViewport } from '../../../../hooks/useViewport';
+import navCurve from '../../../../content/images/nav-curve.svg';
+import Logo from '../../../atoms/Logo/Logo';
+import Icon from '../../../atoms/Icon/Icon';
+import Navbar from '../';
+export interface INavbarProps {
+  /**
```

Comment:
Also we are meant to be moving away from hungarian notation now, right?

Reply:
Did we say what we favour instead?

Reply:
I will fix this in another PR

Reply:
https://github.com/zopaUK/react-components/pull/428